nifi-commons/nifi-expression-language/src/test/java/org/apache/nifi/attribute/expression/language/TestStandardPreparedQuery.java

```diff
@@ -309,6 +309,18 @@ public void testVariableImpacted() {
         assertTrue(Query.prepare("${anyMatchingAttribute('a.*'):equals('hello')}").getVariableImpact().isImpacted("attr"));
     }
 
+    @Test
+    public void testIsExpressionLanguagePresent() {
+        assertFalse(Query.prepare("value").isExpressionLanguagePresent());
+        assertFalse(Query.prepare("").isExpressionLanguagePresent());
+
```

Comment:
I would suggest to add the following additional test statements: 

       assertFalse(Query.prepare("${}").isExpressionLanguagePresent());
        assertFalse(Query.prepare("#{}").isExpressionLanguagePresent());

        assertTrue(Query.prepare("${${xx}}").isExpressionLanguagePresent());
        assertTrue(Query.prepare("${audience}${comma}${question}").isExpressionLanguagePresent());
        assertTrue(Query.prepare("#{param}").isExpressionLanguagePresent());

Reply:
@sjyang18 Thanks for your review.
I added further test cases based on your suggestions.